skills/repo_test.go

```diff
@@ -691,3 +705,38 @@ func (s *RepositorySuite) TestGrantedByReturnsCorrectData(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(s.testRepo.GrantedBy(s.skill.Snap), HasLen, 0)
 }
+
+// Tests for LoadBuiltInTypes()
+
+func (s *RepositorySuite) TestLoadBuiltInTypes(c *C) {
+	err := LoadBuiltInTypes(s.emptyRepo)
```

Comment:
No need to change this now, but in the future it would be nicer to actually implement this as a separate branch that was a pre-req to this one. As it is we're testing to make sure that we have a bug.. we have an exposed function, documented, and tested, doing nothing.